### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ckaminski** [vokac/F2B#1]
Greetings! 

Does this project have a license?

**Comment by vokac** [comment]
No, because I did not spend enough time understanding all the details that comes with different licenses. Personally I prefer widely used/known licenses like Apache 2.0, newer BSD, Mozilla 2.0 or MIT.

If this is an issue I can try to choose...

**Comment by ckaminski** [comment]
Apache and MIT are very similar.  It all depends on whether you want to force contributors to open their code like the GPL, which has it's attractions, or don't care, in which case BSD or MIT could easily suffice.
